### DISCUSSION TITLE
Thank's to repository owner.

### DISCUSSION THREAD

**Discussion by D-Raj-Chakraborty** [BEPb/BEPb#41] [Show and tell]
Hi Andrej Marinchenko,

Sir, I just wanted to reach out and say thank you for creating and maintaining the "BEPb/BEPb" repository. It has been a huge source of motivation for me. Your dedication and hard work are truly inspiring, and I've learned a lot from your README.md file.

Actually I have been watching this repository for last 1 week and realize that how much you put effort on your github_profile-matrics and Metrics.yml file, it's really amazing.

Keep up the amazing work!

Best regards,
Didhiti Raj Chakraborty

**Comment by venelouis** [comment]
yeah very cool

**Reply by BEPb** [reply]
thnks!!!
